stripe/src/test/java/com/stripe/android/view/PaymentMethodsActivityTest.kt

```diff
@@ -169,6 +169,8 @@ class PaymentMethodsActivityTest : BaseViewTest<PaymentMethodsActivity>(PaymentM
 
         val paymentMethodsAdapter =
             recyclerView.adapter as PaymentMethodsAdapter
+
+        paymentMethodsAdapter.selectedPaymentMethodId = paymentMethod.id
         assertEquals(paymentMethod.id, paymentMethodsAdapter.selectedPaymentMethod?.id)
```

Comment:
you can delete these lines

Reply:
Hi Michael. If I delete the assignment the test fails because the selected payment method is null